config/core/configmaps/features.yaml

```diff
@@ -105,6 +105,12 @@ data:
     # See: https://knative.dev/docs/serving/feature-flags/#kubernetes-security-context
     kubernetes.podspec-securitycontext: "disabled"
 
+    # This feature flag allows end-users to add a subset of capabilities on the Pod's SecurityContext.
+    #
+    # When set to "enabled" or "allowed" it allows capabilities to be added to the container.
+    # For a list of possible capabilities, see https://man7.org/linux/man-pages/man7/capabilities.7.html
+    kubernetes.podspec-addcapabilities: "disabled"
```

Comment:
maybe we should make it more obvious with the key that this affects the container security context?

Reply:
Or at a minimum update the description cause there's a distinct Pod & Container SecurityContext

Reply:
renamed key to `kubernetes.containerspec-addcapabilities`